packages/sdk-v2/src/sdk/analytics/index.tsx

```diff
@@ -1,7 +0,0 @@
-export * from './amplitude'
```

Comment:
pls revert all indexes autogenerated by ctix. rename this index.tsx to ts manually. adjust/fix exports manually. do not re-rexport 'defaultConfig' here

Reply:
reverted, adjusted